src/types.ts

```diff
@@ -3,3 +3,26 @@ import { Observable } from 'rxjs'
 export interface Stateful<T> {
   state: Observable<T>
 }
+
+export interface CommonQueryOptions {
+  start: number
+  limit: number
+  orderBy: string
+  orderDirection: 'ASC' | 'DESC'
+}
+
+export interface DaoQueryOptions extends CommonQueryOptions {
```

Comment:
I would prefer to define those in the same file they are used I guess

Reply:
yeah but they are used in different files (not the DaoQueryOptions, but the ProposalQueryOptions and the others are)